homeassistant/components/samsungtv/media_player.py

```diff
@@ -153,69 +171,18 @@ def _power_off_in_progress(self) -> bool:
             and self._end_of_power_off > dt_util.utcnow()
         )
 
-    @property
-    def unique_id(self) -> str | None:
-        """Return the unique ID of the device."""
-        return self._uuid
-
-    @property
-    def name(self) -> str | None:
-        """Return the name of the device."""
-        return self._name
-
-    @property
-    def state(self) -> str | None:
-        """Return the state of the device."""
-        return self._state
-
     @property
     def available(self) -> bool:
         """Return the availability of the device."""
         if self._auth_failed:
             return False
         return (
-            self._state == STATE_ON
+            self.state == STATE_ON
```

Comment:
This is slightly faster

```suggestion
            self._attr_state == STATE_ON
```